sparse/lib/systemd/system/adbd.service

```diff
@@ -1,5 +1,8 @@
 [Unit]
 Description=Android Debug Bridge Daemon
+
 [Service]
 Environment=PATH=/sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
+ExecStartPre=/usr/sbin/adbd-functionfs.sh
 ExecStart=/sbin/adbd
+ExecStopPost=/bin/sleep 1 ; /bin/umount adb
```

Comment:
Does this work with ; ? Also why do we need sleep here? There is kill signal sent to adbd on stop and execstoppost is not executed before that is done, see http://www.freedesktop.org/software/systemd/man/systemd.kill.html